Source/WebCore/platform/graphics/avfoundation/objc/ImageDecoderAVFObjC.mm

```diff
@@ -420,8 +401,13 @@ SampleFlags flags() const override
     m_size = expandedIntSize(m_imageRotationSession->rotatedSize());
 }
 
-bool ImageDecoderAVFObjC::storeSampleBuffer(CMSampleBufferRef sampleBuffer)
+bool ImageDecoderAVFObjC::rawImageFromSampleBuffer(CMSampleBufferRef sampleBuffer, CGImageRef *rawImage)
 {
+    if (!sampleBuffer) {
+        RELEASE_LOG_ERROR(Images, "ImageDecoderAVFObjC::storeSampleBuffer(%p) - sampleBuffer is null", this);
+        return false;
+    }
+
```

Comment:
Please null-check `rawImage`, similarly to `sampleBuffer`.

Reply:
Done.